### ISSUE TITLE
Accessiblity bug - display resolution

### ISSUE THREAD

**Issue by bharathpalaksha** [microsoft/azuredatastudio#25437]
- ADO issue link - [A11y_Azure Cosmos DB Migration for MongoDB_Select or create Azure data base Migration service_Create new pop-up_Resize: "Create Azure data base Migration service" popup is truncated and content also not visible.](https://msdata.visualstudio.com/CosmosDB/_workitems/edit/2829077)
- Azure Data Studio Version: 1.47.0 (user setup)
- Extension: Azure Cosmos DB Migration for MongoDB v2.0.1
- Operating System: Windows 11 23H2
- OS Build: 22631.2715

This accessibility bug is on the extension but we use Azure data studio provided components and we have an issue with createModelViewDialog() . The dialog box which appears is unable to change according to display resolution

Pre-requisite:

- Change the display resolution to 1280*768 and drag the text size slider to 200%

Repro Steps:

1. Open Azure data studio application.
2. Tab to Extensions.
3. Serach for Azure cosmos DB migration for MongoDB extension and install.
4. Tab to connections and click on new connection.
5. Select mongo account login with valid credentials.
6. Press alt+Shift+Fn+F10 keys on the connection and press down arrow keys navigate till manage.
7. Tab to azure cosmos DB migration.
8. Tab to Dashboard tab.
9. Tab to New migration button.
10. Tab to Run a new assessment of your MongoDB server.
11. Fill all the details.
12. Tab to run validation and run the validation.
13. Succuss message will appear.
14. Tab to start "Start assessment button".
15. View assessment results and select database (s) for migration will appear.
16. Tab to data base and select any data base.
17. Specify Azure cosmos DB for MongoDB target Account will appear.
18. Fill all the details and click next button.
19. Migration mapping screen will appear and fill all details and press enter.
20. Select or create Azure data base Migration service will appear.
21. Tab to Create new link under and press enter.
22. Observed that "Create Azure data base Migration service" popup is truncated and content also not visible.

Actual Result:
"Create Azure data base Migration service" popup is truncated and content also not visible.

Expect Result:
"Create Azure data base Migration service" popup should be clearly visible without any truncation, overlapped and overflow

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by kisantia** [comment]
@bharathpalaksha were you able to repro this? There was a similar issue opened, and it was closed when it was unable to be reproduced https://github.com/microsoft/azuredatastudio/issues/24220

**Comment by bharathpalaksha** [comment]
yes, i could reproduce this @kisantia

**Comment by Samhitha66** [comment]
#A11yMAS; #SH_Azure Data Studio_Nov2023_Win32_Dec2023; #SQL Azure Data Studio;

**Comment by laurennat** [comment]
I was not able to reproduce this issue. Can you provide a screenshot of the issue?

**Comment by bharathpalaksha** [comment]
@laurennat agreed, this looks like it's resolved in latest ADS 1.49.1
@Samhitha66 please test again and close this bug

**Comment by PadminiGariki** [comment]
verified the issue in below environment:
Azure Cosmos DB Migration for MongoDB- v3.1.3 Preview
Operating System: Windows 11 23H2
Version: 1.49.1 (user setup)
#Closed; #Regressed:09-09-24;
Issue is fixed. Hence the bug is closed.

**Comment by bharathpalaksha** [comment]
Fixed
